src/main/java/com/iggroup/universityworkshopmw/integration/transformers/MarketDataTransformer.java

```diff
@@ -11,7 +11,7 @@
       return listOfMarkets.stream()
             .map(market -> MarketDto.builder()
                   .marketId(market.getMarketId())
-                  .marketName(market.getMarketName())
+                  .marketName(market.getMarketName().getMarketName())
```

Comment:
could this be changed to something like market.getMarketName.name()

Reply:
yeah, we could do that